modules/custom/openy_system/src/Form/OpenyModulesListForm.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Drupal\openy_system\Form;
+
+use Drupal\system\Form\ModulesListForm;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Provides openy modules installation interface.
+ */
+class OpenyModulesListForm extends ModulesListForm {
+
+  protected $openyPackages = [
+    'OpenY',
+    'OpenY (Experimental)',
```

Comment:
```OpenY Deprecated``` should be added?

Reply:
this is not finished yet, and will be updated in the next version

Reply:
But I see OpenY Deprecated in current PR, so makes sense to add it to this list @ddrozdik